.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest a feature for the project
+labels: feature, incoming
+---
+
+**Is your feature request related to a problem? Please describe.**
+
+A clear and concise description of what the problem is.
+Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+
+A clear and concise description of any alternative solutions or features you've
+considered.
+
+**Additional context**
+
+Add any other context or screenshots about the feature request here.
+
+**Definition of Done**
```

Comment:
Maybe write someting in this section like we do for all other ones?

Reply:
Yeah, but I don't know how to describe it. Will do.

Reply:
added "A clear and concise understanding of expectations that the current feature request must meet in order to be released to users."